http/jsonapi/generator/internal/articles/open-api.json

```diff
@@ -226,6 +226,52 @@
                         }
                     }
                 }
+            },
+            "MapType1": {
+                "type": "object",
+                "additionalProperties": true,
+                "properties": {
+                    "something": {
+                        "type": "string"
+                    }
+                }
+            },
+            "MapType3": {
+                "type": "object",
+                "additionalProperties": {
+                    "$ref": "#/components/schemas/Comment"
+                }
+            },
+            "MapTypeInt": {
+                "type": "object",
+                "additionalProperties": {
+                    "type": "integer",
+                    "format": "int64"
+                }
+            },
+            "MapTypeBool": {
+                "type": "object",
+                "additionalProperties": {
+                    "type": "boolean"
+                }
+            },
+            "MapTypeNumber": {
+                "type": "object",
+                "additionalProperties": {
+                    "type": "number",
+                    "format": "float"
+                }
+            },
+            "MapTypeString": {
+                "type": "object",
+                "additionalProperties": {
+                    "type": "string"
+                },
+                "properties": {
+                    "something": {
+                        "type": "string"
```

Comment:
what is with a date a string that formats a date? can we make use of the function defined here https://github.com/pace/bricks/blob/master/http/jsonapi/generator/generate_helper.go#L24 somehow?

Reply:
this property was added just to test that a warning log is generated and the properties are ignored. It has nothing to do with a date format

Reply:
it should be picked up by the goType func now. So it should be fine. I meant to maybe add another test case.